manifests/noderesourcetopology/cluster-role.yaml

```diff
@@ -32,7 +32,7 @@ subjects:
     namespace: kube-system
 roleRef:
   kind: ClusterRole
-  name: noderesourcetoplogy-handler
+  name: system:kube-scheduler
```

Comment:
I think I know from where this comes from, we experimented with secondary-scheduler-operator which is doing something similar here. Not sure this (= these manifests) is the right place for this change, though

Reply:
The secondary-scheduler-operator is a way of automating the deployment of scheduler plugins but here we want to provide all the artifacts that the user needs to play around with, configure and deploy this scheduler plugin in their cluster. We provide manifests and the steps we have provided in the README to support that. Without these changes, the scheduler plugin is not going to work as expected hence its important to update these manifests to allow the user to deploy NRT scheduler plugin successfully as a secondary scheduler and to deploy a test pod/deployment with this scheduler.

Reply:
This has been removed from the PR now.